validation/telemetry/sync.4.null_device_version.pass.json

```diff
@@ -24,7 +24,7 @@
       {
         "os": null,
         "version": null,
-        "id": "f38b5add46460b49c7373bac4c99f96bfda39af90ed01ce2b4b24957cd5bbeba",
+        "id": "00000000000000000000000000000000",
```

Comment:
I changed this existing test, but I could also create a separate validation case for this if preferred.